### ISSUE TITLE
[SUGGESTION] Add a configurable timeout feature to alertDialog

### ISSUE THREAD

**Issue by eprimex** [overextended/ox_lib#109]
While writing code that asks a player if they would like to do X thing when offered I noticed player could theoretically permanently halt the running of code until the dialog has been canceled.

So my suggestion is to add a configurable timeout for the cancel button so that if the player doesn't click any of the buttons it will automatically press cancel.

Example usage of feature:

`local alert = lib.alertDialog({
	header = "You're alive!",
	content = 'Would you like a free taxi voucher?',
	centered = true,
	cancel = true,
	timeout = 10000
}) --10sec timeout`

**Comment by N-fire** [comment]
Or just a function to close the dialog

**Comment by LukeWasTakenn** [comment]
Implemented close functions in 6a8eb35a0a1d13b63181750a7e653af20b7bba1e.

You can write your own timeout function like so:

```lua
Citizen.SetTimeout(2000, function()
    lib.closeInputDialog()
    print('Input dialog closed')
end)
local input = lib.inputDialog('Dialog title', {'Dialog row'})
print(input)
```

Which will close the dialog after 2 seconds and print `nil`
